src/Jackett.Common/Utils/Clients/HttpWebClient2.cs

```diff
@@ -67,6 +66,8 @@ public override void OnNext(ServerConfig value)
 
         public override void Init()
         {
+            ServicePointManager.DefaultConnectionLimit = 1000;
```

Comment:
This value is huge according to the default value 2 or 10. https://docs.microsoft.com/en-us/dotnet/api/system.net.servicepointmanager.defaultconnectionlimit?view=netframework-4.8
Maybe this is causing some network problems. Let this value as is in the PR, but we have to take a look in the future.

Reply:
I'll make a note of this. Currently this was in all other classes except HttpWebClient2, which is why it's been added to this one.